dataframe/field_config.go

```diff
@@ -0,0 +1,70 @@
+package dataframe
+
+// FieldConfig represents the display properties for a field
+type FieldConfig struct {
+	Title      string `json:"title,omitempty"`
+	Filterable *bool  `json:"filterable,omitempty"`
+
+	// Numeric Options
+	Unit     string `json:"unit,omitempty"`
+	Decimals *int16 `json:"decimals,omitempty"`
+	Min      *int64 `json:"min,omitempty"`
+	Max      *int64 `json:"max,omitempty"`
+
+	// Convert input values into a display string
+	Mappings []*ValueMapping `json:"mappings,omitempty"`
```

Comment:
`[]Something` is more common than `[]*Something`. This is because a slices are are already reference types. Also `[]Something` can be nil.